Gemfile

```diff
@@ -65,7 +65,9 @@ end
 group :test do
   gem "capybara"
   gem "rspec-rails"
+  gem "selenium-webdriver"
```

Comment:
Are there any alternatives to Selenium? I tried to google, but didn't find any viable ones. I guess the slowness comes from Selenium firing up a browser, not really from capybara?

Reply:
(Selenium is probably a great choice, just poking to see if there are alternatives since we have avoided going down this path for speed of test reasons)

Reply:
Yeah the slowness comes from chrome not really capybara

Reply:
Not sure about alternatives Selenium is the only one I ever used